src/skyro/company/Main.java

```diff
@@ -3,6 +3,75 @@
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+        // Задание №1
+        int[] arr = new int[3];
+        arr[0] = 1;
+        arr[1] = 2;
+        arr[2] = 3;
+        for (int i = 0; i < 3; i++) {
```

Comment:
старайтесь не использовать числа по коду, если их можно заменить на значения переменных/полей. В этом случае код будет адаптирован под любые изменения размера массива
```suggestion
        for (int i = 0; i < arr.length; i++) {
```